packages/amplify_auth_cognito/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.0.1-dev.3 (2020-10-08)
+
+### Bug Fixes
+
+* fix(amplify_auth_cognito): throw error in iOS for unconfirmed user signin
+* fix(amplify_auth_cognito): returns AuthError instead of missing method error
+* feat(amplify_auth_cognito): iOS unit tests
+* fix(amplify_auth_cognito): fix example app Podfile
+* feat(amplify_auth_cognito): integration test initial pass
+* fix(amplify_auth_cognito): truncates android resendsignup response
+* Update example podspecs
+* fix(amplify_auth_cognito): event stream destruction
+* chore: unify gradle versions across repo
```

Comment:
Nit: There are some features/chores under the bug fix heading.

Reply:
@Ashish-Nanda True... but the only "features" are unit tests.